html_css/intermediate_css/more-text-styles.md

```diff
@@ -123,15 +123,10 @@ The full snippet is:
 
 You can see more detail and an example in [this CSS Tricks Article](https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/).  (Be ready to go look that article up every time you want to use this.)
 
```

Comment:
I do not believe we want to remove empty Additional Resource sections out right this moment. We can remove the empty bullet. Coming soon I believe there will be information listed that encourages adding additional resources, especially if they are blank.

```suggestion

### Additional Resources
This section contains helpful links to other content. It isn't required, so consider it supplemental for if you need to dive deeper into something

```